slurm-prepilogue.spec

```diff
@@ -54,12 +55,15 @@ rm -rf %{buildroot}
 /usr/libexec/slurm/prolog/checkpaths.sh
 /usr/libexec/slurm/prolog/checkpaths_stat.sh
 /usr/libexec/slurm/prolog/functions.sh
+/usr/libexec/slurm/prolog/epilog.sh
```

Comment:
you installed it in the epilog dir on line 45, so replace prolog by epilog here

Reply:
Nice catch, thx.